Converters/Int32Converter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace InvestingOak.Converters
+{
+    public class Int32Converter : JsonConverter<int>
+    {
+        public override int Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType == JsonTokenType.String)
+                if (int.TryParse(reader.GetString(), out int d))
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/InvestingOak/InvestingOak/pullRequest?prid=6348574)